tests/test_models.py

```diff
@@ -374,3 +371,239 @@ def test_timdex_record_not_a_list_raises_error(timdex_record_required_fields):
         match="'dates' must be <class 'list'>",
     ):
         timdex_record_required_fields.dates = "test"
+
+
+def test_timdex_record_dedupe_alternate_titles(timdex_record_required_fields):
+    timdex_record_required_fields.alternate_titles = [
+        timdex.AlternateTitle(value="My Octopus Teacher"),
+        timdex.AlternateTitle(value="My Octopus Teacher"),
+    ]
+    assert timdex_record_required_fields.alternate_titles == [
+        timdex.AlternateTitle(value="My Octopus Teacher")
+    ]
```

Comment:
In the assertions, lists are used because list _allow_ duplicates; with these assertions, the test confirm that a _single instance_ of an object remains in the list **after** the converter function is applied.